docerina-ui/src/component/layout.js

```diff
@@ -162,7 +162,7 @@ const Layout = (props) => {
                                     <div className="ui breadcrumb">
                                         {(props.match.params.orgName == "ballerina") && !props.match.params.moduleName.startsWith("lang.") &&
                                             <>
-                                                <Link to="/#stdlibs" className="section">Standard Library</Link>
+                                                <Link to="/#stdlibs" className="section">Ballerina library</Link>
```

Comment:
whats the recommended way to rename the `stdlib` word usages @praneesha? It seems there's one usage in here

Reply:
The recommendations are described in https://github.com/ballerina-platform/ballerina-dev-website/pull/7782#issuecomment-1723704068.